sycl/test-e2e/AOT/double.cpp

```diff
@@ -2,9 +2,8 @@
 // using fp64 can be compiled AOT.
 
 // REQUIRES: ocloc, opencl-aot, any-device-is-cpu
-// REQUIRES: build-and-run-mode
 // RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_tgllp -o %t.tgllp.out %s
```

Comment:
I think I'd prefer to use `%{run-aux}` on all build lines in most cases if we need it at one.